pkg/build/strategies/sti/sti.go

```diff
@@ -648,3 +636,12 @@ func includes(arr []string, str string) bool {
 	}
 	return false
 }
+
+func firstNonEmpty(args ...string) string {
+	for _, value := range args {
+		if len(value) > 0 {
```

Comment:
I still don't get why our "standard" of not writing `value != ""`. I see @php-coder had to carefully change this from the example he pointed to in https://github.com/golang/go/issues/14423.